packages/framework-integration-tests/src/sensor/health/application-add-booster-database-children-health-indicator.ts

```diff
@@ -0,0 +1,26 @@
+import { HealthEndpoint } from '@boostercloud/framework-core'
+import {
+  BoosterConfig,
+  HealthIndicatorResult,
+  HealthIndicatorMetadata,
+  HealthStatus,
+  BOOSTER_HEALTH_INDICATORS_IDS,
+} from '@boostercloud/framework-types'
+
+@HealthEndpoint({
+  id: `${BOOSTER_HEALTH_INDICATORS_IDS.DATABASE}/myApplication`,
+  name: 'Indicator added to the Booster Database indicator through My Application',
```

Comment:
This confused me.... So you can add sensors... to specific parent sensors? Curious!

Reply:
Yes, for instance, if you have a Rocket that a new database container and want to add this information to the database health indicator